api/src/main/java/com/codeforcommunity/rest/subrouter/AuthRouter.java

```diff
@@ -59,6 +61,17 @@ private void registerLogoutUser(Router router) {
     logoutUserRoute.handler(this::handleDeleteLogoutUser);
   }
 
+  private void registerVerifySecretKey(Router router) {
+    Route verifySecretKeyRoute = router.get("/verify/:secret_key");
+    verifySecretKeyRoute.handler(this::handleVerifySecretKey);
+  }
+
+  private void registerCreateSecretKey(Router router) {
+    Route createSecretKeyRoute = router.get("/create_secret/:user_id");
+    ((Route) createSecretKeyRoute).handler(this::createSecretKey);
```

Comment:
Why are you casting this?

Reply:
idk i swear that wasn't me

Reply:
it probably was but i have no idea why it's there